commitizen/config/__init__.py

```diff
@@ -0,0 +1,67 @@
+import os
+import warnings
+from pathlib import Path
+
+from commitizen import defaults
+from .base_config import BaseConfig
+from .toml_config import TomlConfig
+from .ini_config import IniConfig
+
+
+def load_global_conf() -> dict:
+    home = str(Path.home())
+    global_cfg = os.path.join(home, ".cz")
+    if not os.path.exists(global_cfg):
+        return None
+
+    # global conf doesnt make sense with commitizen bump
+    # so I'm deprecating it and won't test it
+    message = (
+        "Global conf will be deprecated in next major version. "
+        "Use per project configuration. "
+        "Remove '~/.cz' file from your conf folder."
+    )
+    warnings.simplefilter("always", DeprecationWarning)
+    warnings.warn(message, category=DeprecationWarning)
+
+    with open(global_cfg, "r") as f:
+        data = f.read()
+
+    conf = IniConfig(data)
+    return conf
+
+
+def read_cfg() -> dict:
```

Comment:
This is also not dict. But I'm not sure which one is instead, I guess it's something like `typing.Type[BaseConfing]`

Reply:
Done.